docs/content/docs/testing/errors.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Common Fluid Framework Errors
+menuPosition: 5
+---
+
+This article lists the most common errors Fluid developers will encounter. The first steps to resolve the errors are listed. If you get an error that is not included here, please [let us know with a GitHub issue](https://github.com/microsoft/FluidFramework/issues) and we’ll update both the documentation and our guidance. 
+
+### **`<X>` is not a constructor** 
+
+[A generic JavaScript exception](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_a_constructor)
+
+**Steps to take** : Investigate the root cause in your application. It’s likely that there is a high-level catch statement around the container operations and the underlying problem is in those operations.  
+
+### **`0x26d`**
+
+The ops fetched from service do not start from the requested first op. 
+
+**Steps to take**: If the issue persists, report this issue to the Fluid team by filing a [GitHub issue](https://github.com/microsoft/FluidFramework/issues). 
+
+### **`0x26f`**
+
+The ops fetched from the service are either out of order or duplicated. 
+
+**Steps to take**: If the issue persists, report this issue to the Fluid team by filing a [GitHub issue](https://github.com/microsoft/FluidFramework/issues). 
+
+### **`0x589` / Local data store detected in attaching state while running GC**
+
+A data store was created in the summarizer client and is in “connecting” state when the summary runs. This leads to summary failures and can eventually lead to document corruption if it happens consistently.  
+
+**Steps to take**: Data stores should never be created in the summarizer client because there is no user interaction. Determine which data store is causing the issue. Use the preceding DataStoreCreatedInSummarizer error. It has the properties fullPackageName and fluidDataStoreId. Those properties identify the data store that was created in summarizer. 
+
+### **BlobOnlyStorage not implemented method used**	 
+
+A call to storage is made before the container is attached. 
+
+**Steps to take**: This is a known issue in the Fluid Framework. It is being investigated and this article will be updated when a fix is deployed. 
+
+### **Cannot read properties of undefined (reading 'x')** 
```

Comment:
Do you really want to document basic JavaScript errors here? If so, how do you decide which ones? Is there something special about this one that makes you want to document it in an article about Fluid errors?

Reply:
This was captured from the frequency of errors we've observed in our telemetry. I don't think this belongs in the public documentation, though..